src/Avalonia.Controls/Flyouts/PopupFlyoutBase.cs

```diff
@@ -47,6 +47,13 @@ public abstract class PopupFlyoutBase : FlyoutBase, IPopupHostProvider
         public static readonly StyledProperty<IInputElement?> OverlayInputPassThroughElementProperty =
             Popup.OverlayInputPassThroughElementProperty.AddOwner<PopupFlyoutBase>();
         
+        /// <summary>
+        /// Defines the <see cref="OverlayInputPassThroughElement"/> property
```

Comment:
Wrong property in the doc summary, should be `PlacementConstraintAdjustment`

Reply:
copy pasta issue 🙈